cli/cmd/component-apply.go

```diff
@@ -76,12 +84,15 @@ func runApply(cmd *cobra.Command, args []string) {
 	kubeconfig, err := getKubeconfig(contextLogger, lokoConfig, false)
 	if err != nil {
 		contextLogger.Debugf("Error in finding kubeconfig file: %s", err)
-		contextLogger.Fatal("Suitable kubeconfig file not found. Did you run 'lokoctl cluster apply' ?")
+
```

Comment:
Keeping the `contextLogger.Debugf` around does not solve the problem as it is mentioned in the commit that it is removing dependency on logger.

Reply:
True, I should change the commit message, it's copy pasta, thanks.

Reply:
Done.